wazo_dird/plugin_helpers/tests/test_confd_client_registry.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2019 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from unittest import TestCase
+
+from mock import Mock, patch
+from copy import deepcopy
+
+from ..confd_client_registry import _Registry
+
+
+SOURCE_CONFIG = {
+    'uuid': '9d3f8211-1217-4461-9d0e-ce5e6ddedf2f',
+    'tenant_uuid': 'f5a28c60-acb7-4dcb-9108-0a68c426f9ce',
+    'auth': {
+        'key_file': 'key.yml',
+    },
+    'confd': {},
+}
+
+
+@patch('wazo_dird.plugin_helpers.confd_client_registry.AuthClient')
```

Comment:
```suggestion
@patch('wazo_dird.plugin_helpers.confd_client_registry.AuthClient', Mock())
```
And you will be able to remove unused param in your test methods